go/mysql/conn.go

```diff
@@ -781,8 +833,16 @@ func (c *Conn) handleNextCommand(handler Handler) error {
 		c.startWriterBuffering()
 
 		queryStart := time.Now()
-		query := c.parseComQuery(data)
+
+
 		c.recycleReadPacket()
+		query := c.parseComQuery(data)
+		//if strings.Contains(strings.ToLower(query), "local"){
```

Comment:
How are you actually going to check this? You have to parse the query, right?

Reply:
Yea I move the actual call to go-mysql-server